idea/idea-core/src/org/jetbrains/kotlin/idea/core/script/ScriptConfigurationManager.kt

```diff
@@ -108,6 +108,9 @@ interface ScriptConfigurationManager {
     fun getAllScriptDependenciesSources(): List<VirtualFile>
 
     companion object {
+        fun getAlreadyInitializedInstance(project: Project): ScriptConfigurationManager? =
+            ServiceManager.getServiceIfCreated(project, ScriptConfigurationManager::class.java)
```

Comment:
In Kotlin you can use `serviceIfCreated<ScriptConfigurationManager>()`.

Reply:
Thanks! Fixed

Reply:
As I remember this method isn't present in 192 (or even 193), so we'll need a bunch file then

Reply:
Yep, it is:
https://buildserver.labs.intellij.net/viewLog.html?buildId=79223509&tab=buildResultsDiv&buildTypeId=Kotlin_KotlinDev_IdePlugin_IdeaUltimate_192

Will revert then